docs/transcription-project-guide/working-with-data.md

```diff
@@ -85,7 +114,47 @@ This is what the task input area looks like for this specific example (i.e. what
 
 This is the data export format for three dropdown tasks, from the project _Shadows on Stone: Identifying Sing Sing's Incarcerated_, presented together in the same workflow step:
 
-`{"task":"T52","task_label":null,"value":[{"task":"T15","value":[{"select_label":"Sentenced (month)","option":true,"value":3,"label":"3 - March"}]},{"task":"T16","value":[{"select_label":"Sentenced (day)","option":true,"value":7,"label":"7"}]},{"task":"T17","value":[{"select_label":"Sentenced (year)","option":true,"value":1898,"label":"1898"}]}]}`
+``` json
+{
+  "task":"T52",
+  "task_label":null,
+  "value":[
+    {
+      "task":"T15",
+      "value":[
+        {
+          "select_label":"Sentenced (month)",
+          "option":true,
+          "value":3,
+          "label":"3 - March"
+        }
+      ]
+    },
```

Comment:
This is an example of a nested task annotation, from PFE, where the value of one task contains another task. It might be worth mentioning that nested tasks are deprecated.